test/new-e2e/system-probe/config/vmconfig-x64.json

```diff
@@ -1,7 +1,99 @@
 {
     "vmsets": [
         {
-            "name": "distro_x86_64",
+            "name": "no_tracer_distro_x86_64",
+            "recipe": "distro-x86_64",
+            "arch": "x86_64",
+            "console_type": "file",
+            "kernels": [
+                {
+                    "dir": "xenial-server-cloudimg-amd64.qcow2",
```

Comment:
Is `dir` a necessary part of this file, or could we derive it from the filename of the `image_source`?

Reply:
It is not necessary. It's an artifact from when we were doing this a little different before.
However, the change has to be made in test-infra-definitions to remove this.